### DISCUSSION TITLE
Mail alerts for Wazuh and Zeek

### DISCUSSION THREAD

**Discussion by juddcbrown** [Security-Onion-Solutions/securityonion#3574] [General]
Greetings,

I have set up the mail configs for Zeek and Wazuh as specified in the documentation (/opt/so/conf/wazuh/ossec.conf and ../zeek/zeekctl.cfg).

Postfix is running fine, with all mail to root being forwarded to my email address via /etc/aliases.

Problem is I'm not seeing any info from either app. I see an occasional Cron email, but nothing from Wazuh or Zeek.

Any thoughts on how I've gone wrong?

Thanks!

**Comment by juddcbrown** [comment]
![Opt_So_Permissions](https://user-images.githubusercontent.com/33126910/112015609-f8c3bf80-8af9-11eb-97ba-760b3ebdd8b2.JPG)
![Ossec_conf](https://user-images.githubusercontent.com/33126910/112015637-feb9a080-8af9-11eb-8fc2-bd709e73862b.JPG)

**Comment by weslambert** [comment]
I think the issue here might be the fact that these services are running in their own Docker container, and you are specifying the SMTP server as localhost (which would be interpreted as local to the container).  You might try the hostname or IP of the box itself to see if it works better for you.

**Reply by juddcbrown** [reply]
Thanks Wes. Would the container have its own mail log file? I don't see any logs written to /var/log/maillog other than postfix testing (mail -s Test root).

**Reply by weslambert** [reply]
I'd have to test -- did you try changing the SMTP server address to point to the host?

**Comment by juddcbrown** [comment]
Wes, that did fix it. Thanks much!

On Fri, Mar 26, 2021 at 15:32 weslambert ***@***.***> wrote:

> I'd have to test -- did you try changing the SMTP server address to point
> to the host?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/3574#discussioncomment-535999>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH4XT7SFRYEESGD257V6E6DTFTVO5ANCNFSM4ZTKOINA>
> .
>
-- 
Judd Brown
